graphics/pixelformat.h

```diff
@@ -28,18 +28,27 @@
 
 namespace Graphics {
 
-/** Template to expand from an n-bit component to an 8-bit component */
+/**
+ * @defgroup graphics_pixelformat Pixel formats
+ * @ingroup graphics
+ *
+ * @brief Structures for managing pixel formats.
+ *
+ * @{
+ */
+
+/** Template to expand from an n-bit component to an 8-bit component. */
 template<int depth>
 struct ColorComponent {
 };
-
+/** @todo Doc required (0-bit element?) */
```

Comment:
I am not quite sure why we have a specialisation for 0.  Maybe it is here for completeness or it might be needed if we have formats without alpha channel for example (although the code in `colorToARGB` seems to be checking for that explicitly to use 0xFF, which makes more sense than 0).

Reply:
I coud use 'Return 0 for an empty color component.' ? Does that make sense?

Reply:
Yes, I think it does.